src/pages/profile/account.tsx

```diff
@@ -61,36 +65,39 @@ const ProfilePage: NextPage = () => {
   } = api.user.update.useMutation({
     onSuccess: () => {
       toast.success(`User updated`);
-      setIsEditing(false);
+      setIsEditingUser(false);
 
       if (!newUserData) throw new Error("newUserData is undefined");
       const { name, email, image } = newUserData;
       const newUserDataForSession = { name, email, image };
 
-      setIsEditing(false);
+      setIsEditingUser(false);
       void updateSession(newUserDataForSession);
     },
     onError: (e) => {
       const errMsg = e.data?.zodError?.fieldErrors.content;
+      toast.error(`Failed to update user`);
       if (errMsg?.[0]) {
-        toast.error(`Failed to update: ${errMsg[0]}`);
+        console.log(errMsg[0]);
       }
-      toast.error(`Failed to update user: ${e.message}`);
+      console.log(e.message);
     },
   });
 
   const { mutate: updatePassword, isLoading: isUpdatingPassword } =
     api.user.updatePassword.useMutation({
       onSuccess: () => {
-        setIsEditing(false);
+        setIsEditingUser(false);
+        setIsEditingPassword(false);
         toast.success(`Password updated`);
       },
       onError: (e) => {
         const errMsg = e.data?.zodError?.fieldErrors.content;
+        toast.error(`Failed to update password`);
         if (errMsg?.[0]) {
-          toast.error(`Failed to update password: ${errMsg[0]}`);
+          console.log(errMsg[0]);
```

Comment:
Whats the point of logging this

Reply:
Same for the above error messages

Reply:
not a very good reason but was thinking of debugging in future ><
thanks for raising it up, id remove it